src/transform-applier.js

```diff
@@ -288,7 +288,7 @@ const _transformPath = function (pathString, transform) {
     return result;
 };
 
-const GRAPHICS_ELEMENTS = ['circle', 'ellipse', 'image', 'line', 'path', 'polygon', 'polyline', 'rect', 'text', 'use'];
```

Comment:
I'm seeing an issue with this change where after it's applied I can't get to the second screen of project 56534898